Automattic/jetpack
# SSO: Add resend invite button

Fixes https://github.com/Automattic/wp-calypso/issues/87716

## Proposed changes:

* Add a button on user's table to allow admins to re-send email invitation. Fixes https://github.com/Automattic/wp-calypso/issues/87716

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply this branch and sync to your atomic
* Navigate to `/wp-admin/users.php`
* If there's not any pending invites, invite a user that you have access to the email
* In the table now, you should see the `Resend invite` option.


<img width="1178" alt="image" src="https://github.com/Automattic/jetpack/assets/2653810/21ac0613-2d4e-40e5-b069-617f8417aba4">
